lib/fizzy/parser_expr.cpp

```diff
@@ -736,8 +730,9 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, FuncIdx f
 
             drop_operand(frame, operand_stack, find_local_type(func_inputs, locals, local_idx));
 
-            push(code.immediates, local_idx);
-            break;
+            code.instructions.push_back(opcode);
```

Comment:
Why use `push_back` for these and not `push(code.instructions, opcode)`?

Reply:
This is the copy of line after the switch.

But I can reverse the question: Why use `push(code.instructions, opcode)` for these when `code.instructions.push_back(opcode)` is enough.

Reply:
Maybe `push` should be renamed to `push_immediate`, and optionally a new helper `push_opcode` added

Reply:
> Maybe push should be renamed to push_immediate, and optionally a new helper push_opcode added

That would be nice for readability, in a separate PR.